alembic/versions/3723d12f8740_add_stripetransaction_table.py

```diff
@@ -0,0 +1,53 @@
+"""Add StripeTransaction table
+
+Revision ID: 3723d12f8740
+Revises: 1ed43776064f
+Create Date: 2017-05-05 02:47:59.225336
+
+"""
+
+
+# revision identifiers, used by Alembic.
+revision = '3723d12f8740'
+down_revision = '1ed43776064f'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+import sideboard.lib.sa
+
+
+try:
+    is_sqlite = op.get_context().dialect.name == 'sqlite'
+except:
+    is_sqlite = False
+
+if is_sqlite:
+    op.get_context().connection.execute('PRAGMA foreign_keys=ON;')
+    utcnow_server_default = "(datetime('now', 'utc'))"
+else:
+    utcnow_server_default = "timezone('utc', current_timestamp)"
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
```

Comment:
This was something @EliAndrewC brought up in another one of my pull requests. Once we verify the migrations are doing what we want we should probably remove these comments from Alembic. They look suspiciously like a TODO that someone forgot about.